paket.dependencies

```diff
@@ -19,15 +19,14 @@ source https://api.nuget.org/v3/index.json
 
 storage: none
 
-clitool dotnet-fake
```

Comment:
Changed this to the dotnet tool manifesto approach as suggested in https://github.com/fsharp/FAKE/issues/2465